### ISSUE TITLE
refact-steroids teste em MacOS.

### ISSUE THREAD

**Issue by chacalfloripa** [HashLoad/boss#65]
Baixei o fonte refact-steroids e fiz.alguns teste no MacOS. 
O build ocorreu com sucesso, mas alguns comandados não estão funcionando. Como por exemplo o install. Este comandados.ainda não foram implementados?

**Comment by viniciussanchez** [comment]
Não... estamos trabalhando nele.. acho que o único comando pronto é o init
